calculated_bot.py

```diff
@@ -274,9 +276,49 @@ def chunks(l, n):
     args = ctx.message.content.split(" ")
 
     if len(args) < 2:
-        await bot.send_message(ctx.message.channel,
+        PastUsers = open("UserIDs.txt", "r+")
+        NotLinked = True
```

Comment:
We should load this at bot startup instead of at every call

Reply:
I put that there to make sure that the text file would continually stay up to date (cause when I was testing, the text file wouldn't update unless I closed then re-opened)

Reply:
![image](https://user-images.githubusercontent.com/44003663/57477679-22222c00-7291-11e9-9eb8-8eb393e871fc.png)